webapp-django/crashstats/symbols/tests/test_views.py

```diff
@@ -39,6 +39,8 @@ def setUp(self):
         super(TestViews, self).setUp()
         self.tmp_dir = tempfile.mkdtemp()
 
+        settings.SYMBOLS_MIME_OVERRIDES['jpeg'] = 'text/plain'
```

Comment:
This is not a good idea because you're potentially affecting the settingss as a global object across other tests. Instead use the context manager which resets the settings object automatically. E.g.
```python
with self.settings(SYMBOLS_MIME_OVERRIDES={'jpeg': 'text/plain'}):
    ...
```